monero/wallet_client_test.go

```diff
@@ -122,10 +121,23 @@ func TestClient_GetAccounts(t *testing.T) {
 }
 
 func TestClient_GetHeight(t *testing.T) {
-	c := NewClient(tests.CreateWalletRPCService(t))
+	c := NewWalletClient(tests.CreateWalletRPCService(t))
 	err := c.CreateWallet("test-wallet", "")
 	require.NoError(t, err)
 	resp, err := c.GetHeight()
 	require.NoError(t, err)
 	require.NotEqual(t, 0, resp)
 }
+
+func TestCallGenerateFromKeys(t *testing.T) {
+	kp, err := mcrypto.GenerateKeys()
+	require.NoError(t, err)
+
+	r, err := rand.Int(rand.Reader, big.NewInt(999))
+	require.NoError(t, err)
+
+	c := NewWalletClient(tests.CreateWalletRPCService(t))
+	err = c.generateFromKeys(kp.SpendKey(), kp.ViewKey(), kp.Address(common.Mainnet),
+		fmt.Sprintf("test-wallet-%d", r), "")
+	require.NoError(t, err)
+}
```

Comment:
The above code isn't new.  It was moved from here:
https://github.com/AthanorLabs/atomic-swap/blob/v0.1.0/monero/rpc_test.go#L16-L27